src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct_V1Compat.txt

```diff
@@ -0,0 +1,9 @@
+﻿#SETUP: PowerFxV1CompatibilityRules
+
+// Untyped blanks are not allowed in the argument that defines the row scope
+>> Distinct(Blank(), true)
```

Comment:
>Distinct(Blank(), true) [](http://example.com/codeflow?start=3&length=23)

This is a breaking change: we won't allow untyped blanks to define the row scope. There isn't a real scenario where this would be useful, besides language semantic tests.